code-challenges.js

```diff
@@ -10,45 +10,154 @@
+// Once I checked to see if the first if statement worked which it did.
+// I then just repeated the same steps for my else if statements.
+// Then last and not least I put the old statement of returning the error just in case the user did not put in a number.
+// I know it probably took me a little longer than most people because I would write a little bit of code and test it out.
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 // --------------------2) Create a function that takes in two arrays of numbers and returns a single array with all the values sorted least to greatest.
 // Use the test variables provided below. Expected output: [-9, -7, 0, 3, 7, 8, 9, 13, 22, 36]
 
-var myNumbers1 = [3, 7, 0, 36, -9]
-var myNumbers2 = [8, -7, 22, 9, 13]
-
-
+// var myNumbers1 = [3, 7, 0, 36, -9]
+// var myNumbers2 = [8, -7, 22, 9, 13]
+// var myNewNumber1 = myNumbers1.slice(4)
+// var myNewNumber2 = myNumbers1.slice(3,4)
+// var myNewNumber3 = myNumbers1.slice(2,3)
+// var myNewNumber4 = myNumbers1.slice(1,2)
+// var myNewNumber5 = myNumbers1.slice(0,1)
+// var myNewNumber6 = myNumbers2.slice(4)
+// var myNewNumber7 = myNumbers2.slice(3,4)
+// var myNewNumber8 = myNumbers2.slice(2,3)
+// var myNewNumber9 = myNumbers2.slice(1,2)
+// var myNewNumber10 = myNumbers2.slice(0,1)
+// var myArray = myNewNumber1.concat(myNewNumber9).concat(myNewNumber3).concat(myNewNumber5).concat(myNewNumber4).concat(myNewNumber10).concat(myNewNumber7).concat(myNewNumber6).concat(myNewNumber8).concat(myNewNumber2)
+// console.log(myArray)
+/////////////////////////////////////////////////////////////////////// Todd's Pseudocode //////////////////////////////////////////////////////////////
+// All I have to say is you do the best You can do with what you have.
+//I took what I had learned during the week and stuck with what I know I understand that it's probably a long drawn out process but I got the job done.
+//I knew that by using the slice method I would be able to chop a single number out of an array so that's what I did I chopped it 10 times for 10 numbers.
+//Then I figured out which number went where and used the concat method ten times to make a new variable called myArray.
+// After I had done all of this I wanted to know if there was a simpler way so I got online and talked to a couple of other coders that I know and this was the solution they gave me.
+// 
+// var myNewNumber = myNumbers1.concat(myNumbers2)
+// // console.log(myNewNumber)
+
+// var myNewArray = myNewNumber.sort(function(a,b){ return a - b});
+// console.log(myNewArray)
+//This is a function that allows you to take an array break it down by numbers. 
+// sort that array then return the assorted numbers.
+//Reading the syllabus we should be learning about these types of functions next week
```

Comment:
You are on point-- and you did a great job of breaking the problem apart and trying to manipulate it in a meaningful way.